.github/workflows/docker.yml

```diff
@@ -0,0 +1,47 @@
+name: Publish Docker Superhero Wallet image
+
+on:
+  push:
+    branches: [ master, develop ]
+    tags:
+      - 'v*'
+  pull_request:
+    branches: [ master, develop ]
+
+jobs:
+  main:
+    name: Wallet Build/Test/Docker
```

Comment:
I guess its only docker right?

Reply:
Depends on how we look on it , inside the docker we have buid, test and we have docker build and push